formats/utilityClass/utilityClass.test.js

```diff
@@ -59,6 +59,14 @@ const expectedOutput = `/**
 
 .border-color-primary-lightest { border-color: #c3ecd8; }
 
+.hover\\:background-color-primary-lightest:hover { background-color: #c3ecd8; }
```

Comment:
I've never seen a colon used in a class name like this before. so the hover and focus class names look this?

`class="hover:background-color-primary-lightest"`

or

`class="focus:border-width-xs-hd"`